keyboards/gray_studio/think65v3/think65v3.c

```diff
@@ -0,0 +1,3 @@
+// Copyright 2023 Yizhen Liu (@edwardslau)
+// SPDX-License-Identifier: GPL-2.0
+#include "think65v3.h"
```

Comment:
```suggestion
#include "quantum.h"

const rgblight_segment_t PROGMEM my_capslock_layer[] = RGBLIGHT_LAYER_SEGMENTS(
    {0,2, HSV_WHITE}
);     
const rgblight_segment_t* const PROGMEM my_rgb_layers[] = RGBLIGHT_LAYERS_LIST(
    my_capslock_layer
);

void keyboard_post_init_kb(void) {
    // Enable the LED layers
    rgblight_layers = my_rgb_layers;
}
bool led_update_kb(led_t led_state) {
    if (!led_update_kb(led_state)) {return false;}
    rgblight_set_layer_state(0, led_state.caps_lock);
    return true;
}
```
* Should be renamed to `quantum.h` after the deletion of `think65v3.h`, assuming the matrix positions are moved to `info.json`.
* Move code from `keymap.c` to `think65v3.c`